site/app/libraries/database/DatabaseQueries.php

```diff
@@ -5040,6 +5040,46 @@ private function deleteGradedComponent(GradedComponent $graded_component) {
         $this->course_db->query($query, $params);
     }
 
+    private function updateOverallComments(TaGradedGradeable $ta_graded_gradeable) {
+        foreach ($ta_graded_gradeable->getOverallComment() as $user_id => $comment) {
+            $this->updateOverallComment($ta_graded_gradeable, $user_id, $comment);
+        }
+    }
+
+    private function updateOverallComment(TaGradedGradeable $ta_graded_gradeable, $grader_id, $comment) {
```

Comment:
Switch to `TaGradedGradeable $ta_graded_gradeable, $grader_id, $comment` to `TaGradedGradeable $ta_graded_gradeable, $comment, $grader_id` to be consistent in the other function call parameter ordering.

Reply:
Updated